### PR TITLE
Setup for ui given

### PR DESCRIPTION
**Pull request by phil-davis**
This is WIP. First remove the AdminLogin tag and use "Given" statements when we need an admin login.

### CODE REVIEW

**Comment by phil-davis** [comment]
Note: iAmLoggedInAsUsingThePassword did the same/similar thing to iLoginWithUsernameAndPassword
iAmLoggedInAsUsingThePassword was not used yet. And it seems OK to say either:
"Given I login with username :username and password :password"
or
"When I login with username :username and password :password"

so iLoginWithUsernameAndPassword should be sufficient.

**Comment by phil-davis** [comment]
I think there is no guarantee about which order BeforeScenario steps will run in. Probably it is alpabetical order, and so FeatureContext BeforeScenario  is run before LoginContext BeforeScenario. And that did ensure that the regular users details in FeatureContext were already loaded by the time LoginContext BeforeScenario came along asking for them. But I suspect it would not work for AaaContext named alphabetically before FeatureContext. So I have removed that possibility - every time something in LoginContext wants to know RegularUserNane or RegularUserPassword, it can call the method in FeatureContext to find out.

File: tests/ui/features/bootstrap/UsersContext.php [hunk 1]
```diff
@@ -34,11 +35,14 @@
  */
 class UsersContext extends RawMinkContext implements Context
 {
+	
 	private $loginPage;
-	private $filesPage;
 	private $usersPage;
-	public function __construct(LoginPage $loginPage, UsersPage $usersPage)
+	private $owncloudPage;
```
Comment (individual-it) [inline tests/ui/features/bootstrap/UsersContext.php]: $owncloudPage is not used in that context

**Comment by phil-davis** [comment]
This exact code has been made into just 2 functional commits (for skeleton files and "the rest") in ownCloud core repo and a PR is in progress there.
Work finished here.

**Pull request closed without merging**